src/Converter/StringToArray.php

```diff
@@ -33,9 +33,9 @@ public function __construct($str, $delim = "")
      * @param int|null $limit
      * @return array
      */
-    public function stringToArray($limit)
+    public function stringToArray($limit = null)
     {
-        if (is_null($this->delim)) {
+        if (empty($this->delim)) {
```

Comment:
If string `"0"` or int `0` is used as a delimiter will this still work as intended? `empty(0)` and `empty("0")` return true, which would cause no delimiter to be used.